test/access-token.test.js

```diff
@@ -15,6 +15,10 @@ var ds = loopback.createDataSource({ connector: loopback.Memory });
 Token.attachTo(ds);
 var ACL = loopback.ACL;
 
+beforeEach(function() {
+  this.app = app = loopback();
```

Comment:
LoopBack is using eslint-config-loopback@^1.0.0, the use strict check was added later on -- we need to update this project to use ^4.0.0